loadtest/clan.go

```diff
@@ -257,3 +258,37 @@ func (app *App) getSearchClansOperation() operation {
 		},
 	}
 }
+
+func (app *App) getRetrieveClansSummariesOperation() operation {
+	operationKey := "retrieveClansSummaries"
+	app.setOperationProbabilityConfigDefault(operationKey, 1)
+	return operation{
+		probability: app.getOperationProbabilityConfig(operationKey),
+		canExecute: func() (bool, error) {
+			return true, nil
```

Comment:
I think the main purpose of this test is to stress this route and require `ClansSummaries` only when it has a significant amount of clans. That is, I prefer to only execute this route when a have a minimum amount of clans.

Reply:
I agree, but in this case I've decided to use the shared clans list, so the `retrieveClansSummaries` route can be stressed right away: shared clans are loaded on the load test app startup from a config map.

Reply:
I forgot shared clans are loaded from a config map.